### ISSUE TITLE
Fix S5122 FN: Support raw string literals interpolation

### ISSUE THREAD

**Issue by cristian-ambrosini-sonarsource** [SonarSource/sonar-dotnet#6383]
### Description

FN for S5122 (PermissiveCors).

Using non constant string interpolation in C#11 raw string literals, the rule is not raising an issue.

### Repro steps

```csharp
           const string part1 = "Access-Control";
           const string part2 = "Allow-Origin";
           Response.Headers.Add($$"""{{part1}}-{{part2}}""", "*"); // FN
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
